app/graphql/client.tsx

```diff
@@ -102,7 +129,70 @@ const GaloyClient: React.FC<{ children: React.ReactNode }> = ({ children }) => {
         }),
       )
 
-      const splitLink = split(
+      const errorLink = onError(({ graphQLErrors, networkError: networkErrorRaw }) => {
```

Comment:
Has this been tested at all? My concern is that since the error link is executed for each query, if we make multiple queries at the same time, we will see multiple instances of the same toast issue. In the past, it seems the checks for network errors were psuedo-batched by being processed all at once.

Reply:
I need to make some testing around this. 

Where is batching done by the previous code?

Reply:
Previously the check for errors only ran once after every render with the `useEffect`, so if multiple errors came back in between renders it would only run once.

Reply:
I'm not sure how common that is however. Also not sure if the retry logic here will trigger the error link foreach retry or just once.

Reply:
so I've tested it locally.

basically from my test: the old code didn't work anymore, or at least even if the server was stopped, there was no error showing up.

with the new code, the toast is showing up. It' not reappearing in an annoying way. if you move screen, it may reappear if you deeper in the flow and are loader new graphql queries that are not watched yet. if you go "back there" is no error message.

Reply:
Where does `setNetworkError` get called to clear the error after a toast has been shown?